pkg/cloud/cloud.go

```diff
@@ -392,7 +399,17 @@ func (c *cloud) CreateDisk(ctx context.Context, volumeName string, diskOptions *
 	}
 
 	outpostArn := aws.StringValue(response.OutpostArn)
-
+	var resources []*string
+	if util.IsSBE(zone) {
+		requestTagsInput := &ec2.CreateTagsInput{
+			Resources: append(resources, &volumeID),
+			Tags:      tags,
+		}
+		_, err := c.ec2.CreateTagsWithContext(ctx, requestTagsInput)
+		if err != nil {
+			return nil, fmt.Errorf("could not attach tags to volume: %v. %v", volumeID, err)
```

Comment:
The volume has already been created. You will need to delete the volume here, otherwise you will leak resources.

Reply:
Ack, made changes.